bedrock/newsletter/templates/newsletter/recovery.html

```diff
@@ -1,41 +1,73 @@
-{% extends 'newsletter/base.html' %}
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends 'newsletter/base.html' %}}
+
+{% from "macros-protocol.html" import hero with context %}
 
 {% set_lang_files "mozorg/newsletters" %}
 
 {# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
 {% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
 
-{% block body_id %}newsletter-recovery{% endblock body_id %}
-
 {% block page_title %}{{ _('Newsletter email recovery') }}{% endblock page_title %}
 
 {% block content %}
+<main role="main" class="mzp-l-content">
   {% if form %}
-    {% block main_feature %}
-      <div id="main-feature">
-        <h1>{{ _('Manage your <span>Newsletter Subscriptions</span>') }}</h1>
-      </div>
-    {% endblock %}
-    <form method="post" action="{{ url('newsletter.recovery') }}" id="newsletter-recovery-form" class="container billboard">
-      {% if form.non_field_errors() %}
-        <div class="errorlist">
-          {{ form.non_field_errors() }}
-        </div>
-      {% endif %}
+    <div class="mzp-l-content mzp-t-narrow">
```

Comment:
You have a `mzp-l-content` within another `mzp-l-content` here, which is adding extra padding to the content at smaller screen widths.